app/models/reminder_activity.rb

```diff
@@ -1,2 +1,6 @@
 class ReminderActivity < Activity
+  def self.already_reminded_appointment_ids
```

Comment:
Not 100% on whether this would be more at home in `Appointment`.

Originally was just a local var in the `Appointment.needing_reminder` method but got my wrist slapped by Rubocop for that.